### PR TITLE
Add Troubleshoot the "failed reaching server" error

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?
EDU 124

Adds KB on common error related to certs.

### CODE REVIEW

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (flossypurse) [inline kb/how-to-troubleshoot-last-connection-error.md:10]: Could consider flipping the statement:

An expired TLS certification can often cause `failed reaching server: last connection error`

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (flossypurse) [inline kb/how-to-troubleshoot-last-connection-error.md:12]:
```suggestion
This is a troubleshooting guide that shows you how to do the following:
- Check the certification expiration date
- Renew the certification
- Update the server configuration.
```

Which server? What configuration?

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (flossypurse) [inline kb/how-to-troubleshoot-last-connection-error.md:18]: Feels super redundant if the first couple of sentences are done well.

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (flossypurse) [inline kb/how-to-troubleshoot-last-connection-error.md:56]:
```suggestion
### Update the TLS certification in the server configuration
```

In Cloud?

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (djmagee) [inline kb/how-to-troubleshoot-last-connection-error.md:51]:
```suggestion
If you are using a self-signed certificate or don't have an existing infrastructure, you can generate a new certificate using OpenSSL or [certstrap](https://github.com/square/certstrap).
```

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (djmagee) [inline kb/how-to-troubleshoot-last-connection-error.md:64]:
```suggestion
After you update the TLS certification in the server configuration, retry your connection.
```

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (djmagee) [inline kb/how-to-troubleshoot-last-connection-error.md:73]:
```suggestion
The preceding steps should help you troubleshoot the `failed reaching server: last connection error` error caused by an expired TLS certificate.
```

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (djmagee) [inline kb/how-to-troubleshoot-last-connection-error.md:75]:
```suggestion
If this issue persists, verify that the Client you are using to connect to the server is using the correct TLS certification and that the Client requests reach the server after the roles are fully initialized.
If you still need help, [create a support ticket](/cloud/how-to-create-a-ticket-for-temporal-support).
```

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (djmagee) [inline kb/how-to-troubleshoot-last-connection-error.md:62]:
```suggestion
- [Update certificates using Temporal Cloud UI](https://docs.temporal.io/cloud/how-to-manage-certificates-in-temporal-cloud#update-certificates-using-temporal-cloud-ui)
- [Update certificates using tcld](https://docs.temporal.io/cloud/how-to-manage-certificates-in-temporal-cloud#update-certificates-using-tcld)
```

**Review by flossypurse — APPROVED** [review]
Stamping so that once you get SME approval and get an editorial approval you are gtg.

File: kb/how-to-troubleshoot-last-connection-error.md
Comment (antmendoza) [inline kb/how-to-troubleshoot-last-connection-error.md:44]: with tcld customer can list their certificates and the expiration date, not sure if it helps 
e.g `tcld namespace accepted-client-ca list --namespace antonio.a2dd6`

**Pull request merged**
